recipes/bokeh/meta.yaml

```diff
@@ -0,0 +1,81 @@
+{% set name = "bokeh" %}
+{% set version = "0.11.1" %}
+{% set checksum = "34a73d8f68e7d6faaecc9f785cac8e9f76b88a7e7aacf66307076b03b4d3c472" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ checksum }}
+
+build:
+  number: 0
+  script: python setup.py install
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - six >=1.5.2
+    - requests >=1.2.3
+    - pyyaml >=3.10
+    - python-dateutil >=2.1
+    - jinja2 >=2.7
+    - numpy >=1.7.1
+    - tornado >=4.3
+    - futures >=3.0.3         # [py2k]
+    - pandas
+    - matplotlib
+
+  run:
+    - python
+    - six >=1.5.2
+    - requests >=1.2.3
+    - pyyaml >=3.10
+    - python-dateutil >=2.1
+    - jinja2 >=2.7
+    - numpy >=1.7.1
+    - tornado >=4.3
+    - futures >=3.0.3         # [py2k]
+    - pandas
+    - matplotlib
+
+test:
+  imports:
+    - bokeh
+    - bokeh.application
+    - bokeh.application.handlers
+    - bokeh.charts
+    - bokeh.charts.builders
+    - bokeh.client
+    - bokeh.command
+    - bokeh.command.subcommands
+    - bokeh.core
+    - bokeh.core.compat
+    - bokeh.core.compat.mplexporter
+    - bokeh.core.compat.mplexporter.renderers
+    - bokeh.core.validation
+    - bokeh.models
+    - bokeh.models.widgets
+    - bokeh.plotting
+    - bokeh.sampledata
+    - bokeh.server
+    - bokeh.server.protocol
+    - bokeh.server.protocol.messages
+    - bokeh.server.views
+    - bokeh.sphinxext
+    - bokeh.themes
+    - bokeh.util
+
+about:
+  home: http://github.com/bokeh/bokeh
+  license: BSD 3-Clause
+  summary: Statistical and novel interactive HTML plots for Python
+
+extra:
+  recipe-maintainers:
+    - almarklein
```

Comment:
> Just curious... what does the Node install and then npm install in the bokehjs dir? BokehJS has to be built from coffeescript sources.

If you are getting the source from pypi, there is no need for node steps... All the necessary JS and CSS should be there in the tar.gz package.

In this case they are building from our sources, not from scratch... which I think it is a reasonable approach...